src/style.css

```diff
@@ -1,3 +1,79 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:500&display=swap';
+
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  --main-color: #000;
+  --blue-color: #00acdc;
+
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. Consider adding a generic font family as a fallback after 'Roboto', such as 'sans-serif'.